device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/exceptions/IotHubClientException.java

```diff
@@ -47,7 +47,7 @@ public IotHubClientException(IotHubStatusCode statusCode, Exception nestedExcept
 
     public IotHubClientException(IotHubStatusCode statusCode, String errorMessage, Exception nestedException)
     {
-        super(nestedException);
+        super(errorMessage, nestedException);
```

Comment:
Forgot to pass the error message to the super class alongside the nested exception